en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-reference.md

```diff
@@ -0,0 +1,1058 @@
+            <td>TA secret access key associated with your AWS access key.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>accessKeyId</td>
+            <td>The access key ID associated with your AWS account.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>connectionMaxIdleTime</td>
+            <td>The maximum amount of time(in seconds) that a connection should be allowed to remain open while idle.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>connectionTimeout</td>
+            <td>The amount of time(in seconds) to wait when initially establishing a connection before giving up and timing out.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>connectionTimeToLive</td>
+            <td>The maximum amount of time(in seconds) that a connection should be allowed to remain open, regardless of usage frequency.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>connectionAcquisitionTimeout</td>
+            <td>The amount of time(in seconds) to wait when acquiring a connection from the pool before giving up and timing out.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>socketTimeout</td>
+            <td>The amount of time(in seconds) to wait for data to be transferred over an established, open connection before the connection is timed out.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>apiCallTimeout</td>
+            <td>The amount of time(in seconds) to allow the client to complete the execution of an API call.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>apiCallAttemptTimeout</td>
+            <td>The amount of time(in seconds) to wait for the http request to complete before giving up and timing out.</td>
+            <td>No</td>
+        </tr>
+    </table>
+
+    > **Note**: You can either pass credentials within the `init` configuration or set the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` as environment variables. The AWS SDK uses provider chains to look for AWS credentials in system/user environment variables.
+        
+    To set these environment variables on Linux, macOS, or Unix, use export :
+    
+        export AWS_ACCESS_KEY_ID=AKIXXXXXXXXXXA
+        export AWS_SECRET_ACCESS_KEY=qHZXXXXXXQc4oMQMnAOj+340XXxO2s
+        
+    To set these environment variables on Windows, use set :
+        
+        set AWS_ACCESS_KEY_ID=AKIXXXXXXXXXXA
+        set AWS_SECRET_ACCESS_KEY=qHZXXXXXXQc4oMQMnAOj+340XXxO2s
+    
+    > **Note**: If the application is running in an EC2 instance and credentials are not defined in the init configuration, the credentials will be obtained from the [IAM role](https://docs.amazonaws.cn/en_us/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html) assigned for the Amazon EC2 instance. This option is available only with Amazon SQS connector v2.0.0 and above.
```

Comment:
Please include this note as an [admonition](https://squidfunk.github.io/mkdocs-material/reference/admonitions/?h=) to ensure consistency within the documentation.